wayland-display-core/src/comp/input.rs

```diff
@@ -60,7 +61,7 @@ impl State {
                                 && !modifiers.alt
                                 && !modifiers.logo
                             {
-                                match handle.modified_sym() {
+                                match u32::from(handle.modified_sym()) {
```

Comment:
Would be cleaner to not change this line and match on the new `Keysym` type instead: https://smithay.github.io/smithay/smithay/input/keyboard/struct.Keysym.html